service/servers/oracle/server.go

```diff
@@ -195,6 +196,20 @@ func (os *OracleServer) Prices(ctx context.Context, req *types.QueryPricesReques
 	}
 }
 
+func (os *OracleServer) HandleGetMarketMap(w http.ResponseWriter, _ *http.Request) {
```

Comment:
Instead of writing our own HTTP handler implementations, we usually define our services via [proto](https://github.com/skip-mev/slinky/blob/1657e81e45843d82d1831a18d2795d3fbae8b056/proto/slinky/service/v1/oracle.proto#L14), and use grpc's [grpc-gateway](https://github.com/grpc-ecosystem/grpc-gateway) functionality to spin up a proxy in front of our GRPC server to translate inbound REST requests. The advantage of doing this is that we can 1. define our req/res schema via protobuf, 2. version that schema better, 3. follow idioms defined in the codebase

TLDR: is that I recommend extending the service defined [here](https://github.com/skip-mev/slinky/blob/main/proto/slinky/service/v1/oracle.proto#L14) to include a MarketMap API, implement the service on the `OracleServer` as a GRPC route, and the REST route should be handled for us.

Reply:
cc @Eric-Warehime had mentioned to just make it a http handler -- can switch to gRPC if thats desired! i agree it would make it more uniform w/ rest of code 👍🏻

Reply:
also in favor of just adding an additional RPC method in proto.

Reply:
this is now switched to gRPC, @davidterpay @nivasan1